public/locales/en/menu.json

```diff
@@ -0,0 +1,88 @@
+{
+  "file-menu": {
+    "title": "File",
+    "profile-label": "Profile",
+    "about": {
+      "label": "About",
+      "author": "Author",
+      "license": "License",
+      "url": "URL"
+    },
+    "toggle-console": {
+      "label": "Show Console",
+      "aria-labels": {
+        "checked": "Checked",
+        "unchecked": "Unchecked",
+        "open-console": "Open console",
+        "close-console": "Close console"
+      }
+    },
+    "quit-openspace": {
+      "label": "Quit OpenSpace",
+      "modal": {
+        "title": "Confirm action",
+        "description": "Are you sure you want to quit OpenSpace?",
+        "confirm": "Quit",
+        "cancel": "Cancel"
+      }
+    }
+  },
+  "help-menu": {
+    "title": "Help",
+    "tutorials": "Open Web Tutorials",
+    "getting-started": "Open Getting Started Tour",
+    "send-feedback": "Send Feedback",
+    "open-gui": "Open GUI in Browser",
+    "about": "About",
+    "external-tab-aria-label": "(Opens in new tab)"
+  },
+  "language": {
+    "label": "Language"
+  },
```

Comment:
Do we feel like we need this label? I personally think it can be removed

If you're switching the language, you likely do not understand the language it is written in anyways ;)